src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-snapshots-list/cephfs-subvolume-snapshots-list.component.ts

```diff
@@ -251,6 +251,17 @@ export class CephfsSubvolumeSnapshotsListComponent implements OnInit, OnChanges
     const allGroups = Array.from(this.subvolumeGroupList).map((group) => {
       return { value: group, text: group === '' ? '_nogroup' : group };
     });
+    const nogroupExists = allGroups.some((group) => group.text === '_nogroup');
```

Comment:
Lets create a constant for this in `src/app/shared/constants` , we can have a file created `cephfs.ts`.
DEFAULT_SUBVOLUME_GROUP='_nogroup' and export from there all places, in your NFS PR as well.